mapillary/controller/feature.py

```diff
@@ -35,7 +33,7 @@
 from vt2geojson.tools import vt_bytes_to_geojson
 
 
-def get_map_features_in_shape_controller(geojson: dict, kwargs: dict) -> dict:
+def get_map_features_in_shape_controller(geojson: dict, kwargs: dict) -> str:
```

Comment:
Ideally, we should be returning a `dict` representation of the `geojson`, not a `str`, no?

Reply:
We've been returning GeoJSON as a string so far in different requirements' implementations. We're doing this only to remove the extra leading commas added by default for `dict`s in python. If you try to plot an output GeoJSON directly to [geojson.io](https://geojson.io/#map=15/45.1838/-109.2473), it'll complain about these commas. This was first noticed by @cbeddow and since then I've been returning GeoJSON as a string, then the user would have the option to use `json.loads(geojson)` to use it as a dict. Let's discuss this further and see if we should return the output as a string or a dict!

Reply:
I see. It just seems something that should have handled internally. Converting it to a `dict` sounds like a better solution to me for the future.